### PR TITLE
Adding Swagger for IMDS 2019-02-01

### PR DESCRIPTION
**Pull request by kenchinlai**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Please update the README files [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/imds/data-plane) with this new swagger.

File: specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-02-01/imds.json [hunk 1]
```diff
@@ -0,0 +1,769 @@
...
+                    }   
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Instance"
+                        }
+                    },
+                    "400": {
```
Comment (jhendrixMSFT) [inline specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-02-01/imds.json]: Any status codes added here will cause codegen to treat them as successes, is that intentional?  If not please remove all non-success status codes (comment applies to all operations).

Comment (kenchinlai) [reply specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-02-01/imds.json]: How are these potential responses supposed to be treated? The previous schema version 2018-10-01 had a similar list of potential responses. This new schema version is extremely similar.

Comment (jhendrixMSFT) [reply specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-02-01/imds.json]: Any response code not listed here will be handled by the `default` clause which will treat it as an error.  And if the error response body is ODataV4 compliant the AutoRest codegen will propagate that in the error response (exception in C# etc).

**Comment by kenchinlai** [comment]
Closing this PR. Another member of the team will take over multiple Swagger API updates.

**Pull request closed without merging**